source/hassio/installation.markdown

```diff
@@ -29,6 +29,7 @@ The following will take you through the steps required to install Home Assistant
      - For Hyper-V create a new virtual machine, select "Generation 2", assign it at least 2 GB of memory and select "Connection -> "Your Virtual Switch that is bridged", then "Use an existing virtual hard disk" and select the VHDX file from above, after creation go to "Settings" -> "Security" and deselect "Enable Secure Boot".
      - For KVM create a new virtual machine in `virt-manager`, select "Import existing disk image", provide the path to the QCOW2 image above, choose "Generic Default" for the operating system, assign at least 2 GB memory and 1 vCPU, check the box for "Customize configuration before install" and select your bridge under "Network Selection", then under customization select "Overview" -> "Firmware" -> "UEFI x86_64: ...".
      - For Vmware Workstation create a new virtual machine, select "Custom", make it compatible with the default of Workstation and ESX, Choose "I will install the operating system later", select "Linux" -> "Other Linux 5.x or later kernel 64-bit", give it at least 2 GB RAM and 1vCPU, select "Use Bridged Networking" then "Use an existing virtual disk" and select the VMDK file above, after creation of VM go to "Settings" and "Options" then "Advanced" and select "Firmware type" to "UEFI".
+     - **For VMware ESXi/vSphere/VMware Workstation installs - use the "E1001" or "E1001E" virtual network adapater.**  There are confirmed mDNS/Multicast discovery issues when using VMware's "VMXnet3" virtual network adapter.  
```

Comment:
This should be a list entry. Add it to the "Vmware Workstation" section please.

Reply:
Just discovered that it doesn't appear that VMware Workstation has the E1001 or E1001E vNICs, so this really only applies to ESXi/vSphere installs.  Would it be ok to leave it where it is, like this:

For VMware ESXi/vSphere installs - use the "E1001" or "E1001E" virtual network adapater.**  There are confirmed mDNS/Multicast discovery issues when using VMware's "VMXnet3" virtual network adapter.

Reply:
Then it should stay where it is.